Dockerfile

```diff
@@ -16,4 +16,6 @@ RUN apt-get update \
 
 COPY ./src src/
 
-CMD ["streamlit", "run", "src/app.py"]
+# CMD ["python", "src/app.py"]
```

Comment:
we can remove the comment version afterwards

Reply:
Done.